pom.xml

```diff
@@ -532,8 +532,7 @@
                 <exclude>src/test/resources/**/*</exclude>
                 <exclude>src/test/resources/**/*.css</exclude>
                 <exclude>**/*.jj</exclude>
-                <exclude>src/main/resources/META-INF/services/org.apache.maven.surefire.api.provider.SurefireProvider
-                </exclude>
+                <exclude>src/main/resources/META-INF/services/**</exclude>
```

Comment:
Don't change this POM, it has nothing to do with what you are aiming for. Rather do not make any additional refactoring because it is very hard to separate reasonable and non-reasonable changes. I guess you wanted to make some specific improvement in JUnit5 provider. Do not press CTRL+L because there is again unnecessary changes.

Reply:
Not sure I follow. 

This was an intentional change, otherwise Rat would expect to embed a license inside `[META-INF/services/org.apache.maven.surefire.junitplatform.TestSelectorFactory](https://github.com/apache/maven-surefire/pull/487/files#diff-c7cece3a6cc077f79ed38dae264e97961c801cb107a602888ff19464dbbf4a54)`